app/assets/stylesheets/editor.css

```diff
@@ -160,39 +160,10 @@ version of the editor is released. */
   resize: vertical;
 }
 
-#imagebar {
```

Comment:
Hi @shreyaa-sharmaa -- actually, we'd like to keep the image upload bar as well, so we have both. Will that be possible? Thank you! I'm sorry i missed this.

Reply:
Will do that.